### ISSUE TITLE
Is it possible to backup ubunut system directly to a removable drive

### ISSUE THREAD

**Issue by xiaomeixw** [BluewhaleRobot/systemback#37]
My ubunut system is only 10 gigabytes and I want to back it up to a removable drive

**Comment by randoms** [comment]
No, this is not possible unless you change the code. Systemback will create a tmp directory in home while backup the system. And it will make a copy of system files.  2G is not enough for that.
